solc/CommandLineParser.cpp

```diff
@@ -1112,6 +1122,8 @@ void CommandLineParser::processArgs()
 				m_options.input.allowedDirectories.insert(allowedPath);
 	}
 
+	checkMutuallyExclusive({g_strNoImportCallback, g_strAllowPaths});
+
```

Comment:
I'd check it before validating both options. No point in bugging the user to use them correctly just to then tell him he can't use them both at all :P